src/main.rs

```diff
@@ -423,5 +453,23 @@ impl WidgetTest for Win {
 }
 
 fn main() {
-    Win::run(()).expect("Win::run failed");
+    // Prepare i18n
+    setlocale(LocaleCategory::LcAll, "");
+    bindtextdomain(GETTEXT_PACKAGE, LOCALEDIR);
+    textdomain(GETTEXT_PACKAGE);
+
+    glib::set_application_name("Epic Asset Manager");
+    glib::set_prgname(Some("epic_asset_manager"));
+
+    let res = gio::Resource::load(RESOURCES_FILE).expect("Could not load gresource file");
+    gio::resources_register(&res);
+
+    let uiapp = gtk::Application::new(
+        Some("io.github.achetagames.epic_asset_manager"),
```

Comment:
```suggestion
        Some(config::APP_ID),
```
As the app-id varies between the "stable" & "devel" release